openedx/core/djangoapps/safe_sessions/middleware.py

```diff
@@ -540,6 +576,25 @@ def _verify_user(request, response, userid_in_session):
                     'An unsafe user transition was found. It either needs to be fixed or exempted.\n' +
                     '\n'.join(request.debug_user_changes)
                 )
+
+            if hasattr(request, 'user_id_list') and request.user_id_list:
+                string_user_id_list = [str(user_id) for user_id in request.user_id_list]
+                user_ids_string = ','.join(string_user_id_list)
```

Comment:
[suggestion] These lines can be combined by passing `join` a generator.
```suggestion
                user_ids_string = ','.join(str(user_id) for user_id in request.user_id_list)
```